staff/almu-itur/js-arrays/concat.js

```diff
@@ -0,0 +1,17 @@
+function concat(arr, arr2) {
+    res = [];
```

Comment:
Variable res is not declared.

I don't know if you try to use the demo res, what is not good (you don't need to use a global array because concat returns an array) or you forgot to declare it.

Reply:
You were right, var was not declared, this has been solved now. Thank you for the feedback!